src/client/pages/Stadspas/Stadspas.tsx

```diff
@@ -124,12 +133,13 @@ export default function Stadspas() {
           Bel <a href="tel:020 252 6000">020 252 6000</a> om een nieuwe Stadspas
           aan te vragen.
         </p>
-        {!isLoadingStadspas && WPI_STADSPAS.content?.ownerType !== 'kind' && (
```

Comment:
Werken we niet meer met `ownerType` ?

Reply:
Nee, met de overgang naar stadspassen via ZorgNed kunnen we niet meer achterhalen wat het ownerType is.

Reply:
Kun je de andere onderdelen die hier mee werken dan ook recatoren? 
![Screenshot 2023-04-03 at 10 59 41](https://user-images.githubusercontent.com/1895171/229463018-b4746a68-da77-4fd3-b690-1c9e2cc3baba.png)